### PR TITLE
there!: Zach, Eli, David

### PR DESCRIPTION
**Pull request by DaNish808**

### CODE REVIEW

File: lib/routes/reviews.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            {$set: req.body},
+            {new: true}
+        )
+            .lean()    
+            .then(mongoRes => res.send(mongoRes))
+            .catch(next);
+    })
+
+;
+
```
Comment (Kdasay16) [inline lib/routes/reviews.js]: Clean up extra lines

File: lib/routes/reviewers.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            .catch(next);
+    })
+    .delete('/:id', (req, res, next) => {
+        Reviewer.findByIdAndRemove(req.params.id)
+            .then( mongoRes => {
+                res.send({removed: mongoRes === null ? false : true});
+            })
+            .catch(next);
+    })
+    .patch('/:id', (req, res, next) => { // eslint-disable-line
```
Comment (Kdasay16) [inline lib/routes/reviewers.js]: Why disable this line?

Comment (Zac-L) [reply lib/routes/reviewers.js]: fixed!

File: lib/routes/reviews.js (hunk 1 above)
Comment (Zac-L) [reply lib/routes/reviews.js]: cleaned up in all files now!

also this is our final version now!

File: test/e2e/films.test.js [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+            return request
+                .post(`/api/${dataObj.dob ? 'actors' : 'studios'}`)
+                .send(dataObj);
+        });
+        
+        return Promise.all(saveAllPromises)
+            .then(res => {
+
+                studios = res.slice(0, studioData.length).map(r => r.body);
+                actors = res.slice( -actorData.length).map(r => r.body);
```
Comment (martypdx) [inline test/e2e/films.test.js]: Bit of too clever code, keep things clean and simple:

```js
const saveActors = Promise.all(actorData.map(actor => {
    return request.post('/api/actors').send(actor).then(res => res.body))
}));

const saveStudios = Promise.all(studioData.map(actor => {
    return request.post('/api/studios').send(studio).then(res => res.body));
});

Promise.all([saveActors, saveStudios])
    .then(([actors, studios]) => {
        //...
    });
```

**Pull request merged**